Sources/StreamChat/Models/User.swift

```diff
@@ -81,7 +84,7 @@ public class _ChatUser<ExtraData: UserExtraData> {
         createdAt: Date = .init(),
         updatedAt: Date = .init(),
         lastActiveAt: Date? = nil,
-        teams: [String] = [],
+        teams: [TeamId] = [],
         extraData: ExtraData = .defaultValue
```

Comment:
If TeamDTO includes users and channels, does it make sense for us to only include the TeamId on our domain models?

Reply:
-> Internal discussion on Slack.